sdk/src/main/java/com/google/cloud/dataflow/sdk/runners/DataflowPipelineTranslator.java

```diff
@@ -1051,11 +1052,9 @@ public void translate(
     // IO Translation.
 
     registerTransformTranslator(
-        PubsubIO.Read.Bound.class, new PubsubIOTranslator.ReadTranslator());
+        PubsubUnboundedSource.class, new PubsubIOTranslator.ReadTranslator());
     registerTransformTranslator(
-        DataflowPipelineRunner.StreamingPubsubIOWrite.class,
-        new PubsubIOTranslator.WriteTranslator());
-
+        PubsubUnboundedSink.class, new PubsubIOTranslator.WriteTranslator());
```

Comment:
@peihe 
Pei, I could do with your help here. Despite my efforts these rewrites never seem  to apply when running on Google Cloud Dataflow.